### PR TITLE
initial-commit

### PR DESCRIPTION
**Pull request by Lana-Lannister**
The first draft of music player project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```
Comment (danielnass) [inline index.html:1]: @Lana-Lannister 🚀

File: index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
     <title>The XX music player</title>
   </head>
   <body>
-    <h1>The XX Player</h1>
+    <h1>The XX Musica Player</h1>
```
Comment (danielnass) [inline index.html]: @Lana-Lannister Please use English only

**Review by danielnass — CHANGES_REQUESTED** [review]
@Lana-Lannister see comment

File: index.html (hunk 2 above)
Comment (Lana-Lannister) [reply index.html]: fixed

**Review by danielnass — APPROVED** [review]

**Pull request merged**